### ISSUE TITLE
request: allow members with admin role to access dashboard

### ISSUE THREAD

**Issue by gc** [skyra-project/skyra#614]
Tldr: if member has admin setting role, let them log into dashboard without needing manage server perm.

In skyra,
Mod role: allows you to use mod commands like ban and prune
Admin role: also allows you to configure settings

In other words, the admin role is the settings manager role. However you could be a skyra role admin but not have manage server perms, which means you can't manage the server on the dashboard. This is actually the case for a couple servers I'm in.

Requires:
1. Change KDH fork so DashboardGuild.userCanManage checks for the admin setting role OR the manage server perm.
2. Change whichever KDH routes need to be
3. Frontend shouldn't need any changes I believe
